azext_edge/edge/commands_schema.py

```diff
@@ -0,0 +1,56 @@
+# coding=utf-8
+# ----------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License file in the project root for license information.
+# ----------------------------------------------------------------------------------------------
+
+from typing import Iterable, Optional
+
+from knack.log import get_logger
+
+from .providers.orchestration.resources import SchemaRegistries
+
+logger = get_logger(__name__)
+
+
+def create_registry(
```

Comment:
no update registry? you should be able to update the display name + description (and tags)

Reply:
Later, not really a P0 in this iteration.

Reply:
tis an extra easy win with the low amount of params but ok